save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/FeaturedContests.kt

```diff
@@ -0,0 +1,253 @@
+/**
+ * Preparation for a participation card
+ */
+
+package com.saveourtool.save.frontend.components.views.contests
+
+import com.saveourtool.save.entities.ContestDto
+import com.saveourtool.save.frontend.components.basic.ContestNameProps
+import com.saveourtool.save.frontend.components.basic.showContestEnrollerModal
+import com.saveourtool.save.frontend.components.modal.displayModal
+import com.saveourtool.save.frontend.components.modal.mediumTransparentModalStyle
+import com.saveourtool.save.frontend.externals.fontawesome.faArrowRight
+import com.saveourtool.save.frontend.externals.fontawesome.fontAwesomeIcon
+import com.saveourtool.save.frontend.utils.*
+
+import csstype.*
+import react.ChildrenBuilder
+import react.StateSetter
+import react.VFC
+import react.dom.aria.AriaRole
+import react.dom.html.ButtonType
+import react.dom.html.ReactHTML
+import react.dom.html.ReactHTML.a
+import react.dom.html.ReactHTML.button
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.h3
+import react.dom.html.ReactHTML.i
+import react.dom.html.ReactHTML.img
+import react.dom.html.ReactHTML.p
+import react.dom.html.ReactHTML.span
+import react.dom.html.ReactHTML.strong
+import react.useState
+
+import kotlinx.js.jso
+
+const val INVERT_TO_OPPOSITE = 100
+
+val yourContests = yourContests()
+
+private fun ChildrenBuilder.carouselArrows() {
+    a {
+        style = jso {
+            filter = invert(INVERT_TO_OPPOSITE)
+        }
+        className = ClassName("carousel-control-prev ")
+        href = "#contestCarousel"
+        role = AriaRole.button
+        asDynamic()["data-slide"] = "prev"
+        span {
+            className = ClassName("carousel-control-prev-icon")
+        }
+    }
+    a {
+        style = jso {
+            filter = invert(INVERT_TO_OPPOSITE)
+        }
+        className = ClassName("carousel-control-next")
+        href = "#contestCarousel"
+        role = AriaRole.button
+        asDynamic()["data-slide"] = "next"
+        span {
+            className = ClassName("carousel-control-next-icon")
+        }
+    }
+}
+
+@Suppress("MAGIC_NUMBER")
+private fun ChildrenBuilder.stayTunedImage() {
+    img {
+        className = ClassName("card-img-right flex-auto d-none d-md-block")
+        src = "img/undraw_notify_re_65on.svg"
+        style = jso {
+            width = 24.rem
+        }
+    }
+}
+
+private fun ChildrenBuilder.stayTuned() {
+    div {
+        className = ClassName("card-body d-flex flex-column align-items-start")
+        ReactHTML.strong {
```

Comment:
```suggestion
        strong {
```
and so on in this file

Reply:
thx

Reply:
fixed